scenes/quests/story_quests/stella_and_the_missing_star/0_intro/intro.tscn

```diff
@@ -0,0 +1,122 @@
+[gd_scene load_steps=10 format=4 uid="uid://c1gdct760l86h"]
+
+[ext_resource type="TileSet" uid="uid://07fq3rspk8ia" path="res://scenes/tileset.tres" id="1_2qnws"]
+[ext_resource type="Script" uid="uid://x1mxt6bmei2o" path="res://scenes/menus/intro/components/intro.gd" id="1_o3x1t"]
+[ext_resource type="PackedScene" uid="uid://iu2q66clupc6" path="res://scenes/game_elements/characters/player/player.tscn" id="2_17a4c"]
```

Comment:
TODO: we would need to make the player not respond to input. Three ways to do this:

- Add some "inert" property to disable input
- Use the existing Pause mechanism to pause player input
- Rather than using the Player scene, use a much simpler scene that shares the sprite frames

Reply:
I'm going to follow @JuanFdS in #199 and use a plain AnimatedSprite2D. I'll also use as similar walk-on for the character.